x-pack/plugins/lens/public/shared_components/toolbar_popover.tsx

```diff
@@ -41,15 +42,17 @@ export const ToolbarPopover: React.FunctionComponent<ToolbarPopoverProps> = ({
   isDisabled = false,
   groupPosition,
   buttonDataTestSubj,
+  larger,
 }) => {
   const [open, setOpen] = useState(false);
 
   const iconType: string | IconType = typeof type === 'string' ? typeToIconMap[type] : type;
+  const panelClassName = larger ? 'lnsXyToolbar__popoverLarger' : 'lnsXyToolbar__popover';
```

Comment:
Not sure about this workaround - maybe @MichaelMarcialis has an idea how to improve this.
For context: To show why an option in the popover is disabled, a tooltip icon is added, but it's slightly too large to fit into the same row for the standard width of these popovers, so it has to get enlarged:
<img width="517" alt="Screenshot 2020-11-03 at 15 39 26" src="https://user-images.githubusercontent.com/1508364/97998736-c6cfec00-1dea-11eb-8832-0d8332ef5f35.png">

The only thing I can think of is to put the tooltip on the disabled form element itself instead of an additional icon, then the popover is wide enough and we don't need a separate class. The downside is the missing affordance - does the user know they have to hover the form element to get a reason for it being disabled?

Reply:
Just realized we have something like this in lens already - it looks like this for the colors:
<img width="402" alt="Screenshot 2020-11-03 at 16 54 56" src="https://user-images.githubusercontent.com/1508364/98008964-5a0e1f00-1df5-11eb-9e8a-ffa0e42d1a4b.png">

No strong opinion which route to go but we should make sure to do it the same way in both places

Reply:
> The only thing I can think of is to put the tooltip on the disabled form element itself instead of an additional icon, then the popover is wide enough and we don't need a separate class. The downside is the missing affordance - does the user know they have to hover the form element to get a reason for it being disabled?

Yes, this was the reason of the conditional icon state. I'll update the PR with a better alignment 👍

Reply:
Hey, all. If I'm understanding correctly, and the "missing values" field is one that will **always** be disabled when any visualization other than line and area are selected, my first instinct would be to conditionally remove that form field altogether in those instances. I'm not a fan of showing users disabled form fields where users have little-to-no ability to re-enable (assuming the currently selected visualization type is the desired one). 

For example, if I know I want a bar chart for my visualization, why even present me with a field that I'll never be able to interact with?

Reply:
Updated PR to hide the field when the chart is not of type area or line.
Also improved icon alignment as in the picture above for `Series color`.